### DISCUSSION TITLE
Controlnet problem

### DISCUSSION THREAD

**Discussion by juanman1000** [Mikubill/sd-webui-controlnet#1773] [Q&A] [answered]
greetings community, thanks for your time
I recently installed controlnet in my automatic1111, but I can't get it to influence the resulting images, install the models, select them according to the preprocessor and when generating does not influence the results and generates a final image with the openpose skeleton, I will attach reference images about my parameters, if someone could help me I would be very grateful.
the pc is a 17 39000k, 12 vram nvidia gpu
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/138880985/33de1da5-4b29-4934-aa7a-af9c05d5ad8c)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/138880985/a75b6e3b-2208-4000-9f20-89dae6b597a6)

**Comment by huchenlei** [comment, marked as answer]
Change your control start step to 0 should help with your problem

**Reply by juanman1000** [reply]
worked for me, thanks pal!
